From e171fa997d04159994d19e89a94006a5afb4e47f Mon Sep 17 00:00:00 2001
From: Krzysztof Kotlenga <k.kotlenga@sims.pl>
Date: Thu, 29 Dec 2022 19:14:32 +0100
Subject: [PATCH 541/726] overlays: audremap: Fix setting of the pin function

The brcm,function property must be set in the overlay fragment too -
otherwise the parametrization won't work. At least that's the case for
bcm2711-rpi-cm4.dts which starts with empty properties:

&gpio {
        audio_pins: audio_pins {
                brcm,pins = <>;
                brcm,function = <>;
        };
};

This was broken since a56df85d2f42fd461fdc05f33617141aca5ba465.

Signed-off-by: Krzysztof Kotlenga <k.kotlenga@sims.pl>
---
 arch/arm/boot/dts/overlays/audremap-overlay.dts | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/boot/dts/overlays/audremap-overlay.dts b/arch/arm/boot/dts/overlays/audremap-overlay.dts
index a73cc95934ba..95027c5c8f9e 100644
--- a/arch/arm/boot/dts/overlays/audremap-overlay.dts
+++ b/arch/arm/boot/dts/overlays/audremap-overlay.dts
@@ -8,6 +8,7 @@ fragment@0 {
                 target = <&audio_pins>;
                 frag0: __overlay__ {
                         brcm,pins = <12 13>;
+                        brcm,function = <4>; /* alt0 alt0 */
                 };
         };
 
-- 
2.33.1

